RxCocoa/Common/DelegateProxy.swift

```diff
@@ -225,19 +225,37 @@
         }
 
         override open func responds(to aSelector: Selector!) -> Bool {
-            return super.responds(to: aSelector)
+            if  aSelector == #selector(self.isDelegateProxy(_:)) {
+                return true;
+            }
+
+            let res = super.responds(to: aSelector)
                 || (self._forwardToDelegate?.responds(to: aSelector) ?? false)
                 || (self.voidDelegateMethodsContain(aSelector) && self.hasObservers(selector: aSelector))
+            return res
+        }
+
+        @objc open func isDelegateProxy(_ proxy: AnyObject) -> NSNumber {
+            if self === proxy {
```

Comment:
`return self === proxy`

Reply:
I had tried, but it doesn't work saying bool is not NSNumber while i need a NSNumber

Reply:
![image](https://user-images.githubusercontent.com/32675842/61996280-4b4e9d80-b0c5-11e9-8e90-5ce67954d92d.png)

Reply:
i had tried and i need a nsnumber. i will fix it is another way
![image](https://user-images.githubusercontent.com/32675842/61996329-b8fac980-b0c5-11e9-996a-0d48af560672.png)